File: traductores/unidad5/meta5_1_V2/programa_generado.c
#include <stdio.h>

int main() {
    double n;
    double prom;
    double i;
    double cal;
    
    printf("cuantas calificaciones ");
    scanf("%lf", &n);
    prom = 0;
    i = 0;
    while (i<n) {
        printf("da una calificacion ");
        scanf("%lf", &cal);
        prom = prom+cal;
        i = i+1;
    }
    prom = prom/n;
    if (prom>5) {
        printf("aprobado con :  %f\n", prom);
    }
    if (prom==10) {
        printf("excelente\n");
    }
    return 0;
    }

File: traductores/unidad5/meta5_1_V2/programa_generado.py
# Programa generado desde pseudocódigo

# Variables declaradas
    
n = float(input('cuantas calificaciones'))
prom = 0
i = 0
while i<n:
    cal = float(input('da una calificacion'))
    prom = prom+cal
    i = i+1
prom = prom/n
if prom>5:
    print('aprobado con : ', prom)
if prom==10:
    print('excelente')
